docs/customresourcestate-metrics.md

```diff
@@ -14,6 +14,8 @@ Two flags can be used:
 
 If both flags are provided, the inline configuration will take precedence.
 
+Apart from specifying one of `--custom-resource-state-config-*` flags, you also need to add a custom resource *Kind* in plural form to the list of existing resources in `--resources` flag. If you don't specify `--resources`, then `--custom-resource-state-config-*` and all available default kubernetes objects will be taken into accout by kube-state-metrics.
```

Comment:
```suggestion
In addition to specifying one of `--custom-resource-state-config*` flags, you should also add the custom resource *Kind*s in plural form to the list of exposed resources in the `--resources` flag. If you don't specify `--resources`, then all know custom resources and all available default kubernetes objects will be taken into account by kube-state-metrics.
```

Reply:
Thanks for suggestion. I think first sentence looks better indeed. 

However 2nd sentence is lacking explanation since It doesn't say explicitly who knows about custom resources (k8s itself of KSM?). So I'd like to be more specific in this part to make it clear for the user.

Reply:
That makes sense to me. Feel free to add additional modifications along those lines 👍

Reply:
Made some changes.